src/mame/konami/konppc_jvshost.h

```diff
@@ -0,0 +1,40 @@
+// license:BSD-3-Clause
+// copyright-holders:windyfairy
+#ifndef MAME_KONAMI_KONPPC_JVSHOST_H
+#define MAME_KONAMI_KONPPC_JVSHOST_H
+
+#pragma once
+
+#include "emu.h"
+
+#include "machine/jvshost.h"
+
+class konppc_jvs_host_device : public jvs_host
+{
+public:
+	// construction/destruction
+	konppc_jvs_host_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
+	void read();
+	bool write(uint8_t data);
+
+	DECLARE_READ_LINE_MEMBER( sense );
+
+	auto output_callback() { return output_cb.bind(); }
+
```

Comment:
Since this is a configuration member function, please place it above the “live” read/write member functions as is the MAME convention.